airflow-scheduler/airflow_scheduler.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+airflow_host=${1:-localhost}
+airflow_port=${2:-8080}
+
+scheduler_status=$(curl -k https://"${airflow_host}":"${airflow_port}"/health | jq --raw-output '.scheduler.status')
```

Comment:
Does this `jq` command come with Linux by default?

Reply:
It does not. I've mentioned it as a pre-req in the README.